src/modules/Gradient/info.json

```diff
@@ -1,6 +1,13 @@
 {
     "name": "Gradient",
     "description": "Gives a gradient of the image",
-    "inputs": {},
+    "inputs": {
```

Comment:
Oh i see! OK, well, we can leave this in if you like, but perhaps in the future it's best to do only one concrete thing in each PR, so we have less to review and it's more self-contained. Make sense? Thank you!

Reply:
This is a cool addition to this module though 😄

Reply:
Hehe, this is funny. Will take care for such kind of changes from the next time. 😄 😄